src/styles/cards.scss

```
@@ -0,0 +1,124 @@
+.main__container {
+  display: flex;
+  flex-wrap: wrap;
+  width: (4 * $card-width) + (3 * $card-margin) + (2 * $inside-padding);
+  margin: 0 auto;
+  padding-left: 2px;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: $white;
+}
```

move to the reset file
```suggestion
p {
  margin: 0;
}

a {
  text-decoration: none;
}
```